src/components/command-bar/commands/search/unified-search/components/unified-hits-container/helpers/get-should-render-integrations-tab.ts

```diff
@@ -0,0 +1,31 @@
+import { ProductSlug } from 'types/products'
+
+/**
+ * Pull which products have integrations off of our global config.
+ */
+const PRODUCT_SLUGS_WITH_INTEGRATIONS =
+	__config.dev_dot.product_slugs_with_integrations
+
+/**
+ * Given the current product slug,
+ * Return `true` if the Integrations search results tab should be rendered,
+ * or `false` otherwise.
+ */
+export function getShouldRenderIntegrationsTab(
+	currentProductSlug: ProductSlug
+): boolean {
+	let shouldRenderIntegrationsTab
+	if (PRODUCT_SLUGS_WITH_INTEGRATIONS.length <= 0) {
+		// If no products have integrations, do not render it
+		shouldRenderIntegrationsTab = false
+	} else if (currentProductSlug) {
+		// If there is a product tag, render it if the product has integrations
+		const productHasIntegrations =
+			PRODUCT_SLUGS_WITH_INTEGRATIONS.includes(currentProductSlug)
```

Comment:
Re: [the issue that](https://github.com/hashicorp/dev-portal/pull/1986#issuecomment-1589666460)  @braydenlove brought up. It looks like `packer` isn't in the config integration slugs array so that's likely the source of the issue? 
<img width="1016" alt="Screenshot 2023-06-15 at 10 06 04 AM" src="https://github.com/hashicorp/dev-portal/assets/36613477/01357dbc-2eb4-4ec1-8c2b-64c7be2429ed">

Reply:
☝🏻 not necessary to solve now, just trying to find a source of the unexpected behavior.

Reply:
ah yeah thanks for this - I believe the issue boils down to:
- The Algolia index will contain all the latest integrations records, and return those if they're relevant
- The Dev Dot UI needs to only show integrations results for products that have explicitly been toggled on in `dev-portal` `config`

I think one potential solution here will be to try to apply an additional filter when fetching from Algolia, based on which products have integrations activated... so that even if a matching record for an integration result _exists_ in Algolia, we'll avoid showing it in the UI but also still fetch the full "page" of relevant records from Algolia... rather than say filter them out _after_ making the query, which would kinda work but reduce the number of results shown, perhaps significantly in some cases, and even for queries that might have many more relevant results... anyways 😅  🤔 I think I might need facets here or something... going to explore in a follow up PR!

Reply:
Makes sense! Sounds good to me.